src/test/strict_priorities.c

```diff
@@ -38,6 +38,6 @@ int main(void) {
   main_thread_done = 1;
 
   atomic_puts("EXIT-SUCCESS");
-
+  (void)dummy; /* Avoid -Wunused-but-set-variable build warning. */
```

Comment:
This is probably a real bug in this test. The compiler will delete the above loop if this variable is unused. Could be fixed with either a volatile or some sort of asm memory call.

Reply:
Interesting! I admit I didn't actually do serious analysis of what the test is trying to do here.

If the test does in fact need fixing beyond this warning-silencing-bandaid, I'll leave that to someone who's got some familiarity with rr internals/development/testing (i.e. not me).